### DISCUSSION TITLE
SMILE parsing error

### DISCUSSION THREAD

**Discussion by ivannnnnnnnnn** [rdkit/rdkit#6264] [Development]
Hello! I receive SMILE parse error when try to create molecule from smiles

Code:
`mol = Chem.MolFromSmiles("c1ccc(Sc2cccc3c2-c2nc-3nc3c4c(Sc5ccccc5)cccc4c4nc5nc(nc6c7c(Sc8ccccc8)cccc7c(n2)n6[al+]n43)-c2cccc(Sc3ccccc3)c2-5)cc1")`

Output:
`SMILES Parse Error: syntax error while parsing: c1ccc(Sc2cccc3c2-c2nc-3nc3c4c(Sc5ccccc5)cccc4c4nc5nc(nc6c7c(Sc8ccccc8)cccc7c(n2)n6[al+]n43)-c2cccc(Sc3ccccc3)c2-5)cc1`

What is wrong with this smiles?

I tried to get a molecule from these smiles on these sites and there were no errors

https://pubchem.ncbi.nlm.nih.gov//edit3/index.html
https://www.rcsb.org/chemical-sketch

**Comment by bp-kelley** [comment]
The rdkit doesn't recognize aromatic aluminium:

> Chem.MolFromSmiles("[al+]")

[16:25:05] SMILES Parse Error: syntax error while parsing: [al+]

[16:25:05] SMILES Parse Error: Failed parsing SMILES '[al+]' for input:
'[al+]'


>>> Chem.MolFromSmiles("[Al+]")

<rdkit.Chem.rdchem.Mol object at 0x107c99c10>

**Reply by ivannnnnnnnnn** [reply]
Thank you, it's really solve my problem
